andela/reaction-commerce
# #133002627 admin user should be able to cancel orders

### What does this PR do?
Adds the ability for the admin user to cancel an order.

### Description of Task to be completed?
* Add a React Js button component to the order workflow template.
* Add a Meteor Js DDP method to deal with canceling the order which included the following sub tasks
  * Send out email to the buyers that their orders have been canceled.
  * Offer a refund if their payment has been captured.
  * Restock the inventory.
  * Change the status of the order to canceled.

### How should this be manually tested?
Login as a buyer and make an order. Login into the admin interface for the shop, go to the order page and start processing the order. The cancel button should appear in the admin sidebar that shows details on the orders, click on it and the order should be moved from `processing` to `canceled` and its status changed.

### What are the relevant pivotal tracker stories?
#133002627 Admin user should be able to cancel orders

### Screenshots (if appropriate)
![image](https://cloud.githubusercontent.com/assets/20815290/19965084/8ddf823e-a1c5-11e6-8d0f-ffea1f140f5c.png)
![image](https://cloud.githubusercontent.com/assets/20815290/19965096/9fac7bf2-a1c5-11e6-841a-344b5a2bccf5.png)
